src/components/typography/hoverable.tsx

```diff
@@ -20,10 +21,24 @@ export const HoverableText: FC<HoverableTextProps> = (
   return <Text {...props} variant="title" sx={sx} />;
 };
 
-export const HoverableLink: FC<HoverableLinkProps> = (
-  props: HoverableLinkProps,
-) => {
+export const HoverableLink: FC<HoverableLinkProps> = ({
+  href,
+  target,
+  ...props
+}: HoverableLinkProps) => {
   let { sx } = { ...props };
   sx = { ...sx, ...styles };
-  return <Link {...props} variant="title" sx={sx} />;
+
+  return (
+    <RebassLink
+      as={Link}
+      {...props}
+      // @ts-ignore
+      to={href}
```

Comment:
the `to` prop is rejected, since ts is having hard time to recognize the `as` prop things
I guess the typings in `rebass` could be off too